README.adoc

```diff
@@ -108,6 +108,35 @@ List<Product> found = products.findByNameLike(searchPattern, Order.by(
                                               _Product.id.asc()));
 ----
 
+== Jakarta Data Query Language (JDQL)
+
+Jakarta Data introduces the Jakarta Data Query Language (JDQL), a streamlined query language designed to specify the semantics of query methods within Jakarta Data repositories. Utilizing the `@Query` annotation, JDQL allows developers to define queries straightforwardly and robustly.
+
+JDQL is conceptualized as a subset of the Jakarta Persistence Query Language (JPQL). It inherits its syntax and functionality while being specifically tailored to accommodate the broad spectrum of data storage technologies supported by Jakarta Data. This design approach ensures that JDQL retains the expressiveness of SQL and JPQL yet simplifies its implementation across diverse data stores.
+
+[source,java]
+----
+@Repository
+public interface BookRepository extends BasicRepository<Book, UUID> {
+
+  // Find books with titles matching a specific pattern
+  @Query("where title like :titlePattern")
+  List<Book> booksMatchingTitle(String titlePattern);
+
+  // Select books by a specific author and sort them by title
+  @Query("where author.name = :author order by b.title")
```

Comment:
This query does not define the identifier variable `b` anywhere, and in any case that is not part of JDQL,
```suggestion
  @Query("where author.name = :author order by title")
```